apps/src/sites/studio/pages/components/editorRedux.js

```diff
@@ -0,0 +1,127 @@
+/* global levelKeyList */
```

Comment:
Can we avoid having this as a global? First thought (without looking at the code to much) is to have an initial action that populates the store with it.